composer.json

```diff
@@ -9,13 +9,15 @@
             "EventSauce\\MessageOutbox\\DoctrineOutbox\\": "src/DoctrineOutbox",
             "EventSauce\\IlluminateMessageRepository\\": "src/IlluminateMessageRepository",
             "EventSauce\\DoctrineMessageRepository\\": "src/DoctrineMessageRepository",
-            "EventSauce\\MessageRepositoryTestTooling\\": "src/MessageRepositoryTestTooling"
+            "EventSauce\\MessageRepositoryTestTooling\\": "src/MessageRepositoryTestTooling",
+            "EventSauce\\DynamoDBMessageRepository\\": "src/DynamoDBMessageRepository"
         }
     },
     "require": {
         "eventsauce/eventsauce": "^1.0",
         "eventsauce/backoff": "^1.0",
-        "doctrine/dbal": "^3.1"
+        "doctrine/dbal": "^3.1",
+        "aws/aws-sdk-php": "^3.0"
```

Comment:
Question: Not sure if I should add this here

Reply:
This is ok 👍